src/containers/guest-home-page/how-it-works/HowItWorks.jsx

```diff
@@ -18,10 +18,10 @@ import { styles } from '~/containers/guest-home-page/how-it-works/HowItWorks.sty
 const HowItWorks = () => {
```

Comment:
You created the unit tests for the AppTextArea, but the HowItWorks component would be overwritten by your commit
![image](https://github.com/Space2Study-UA-1156/Client-01/assets/93076661/118be721-ce4d-401e-b414-06740c1df182)


I think you should adjust your commits

Reply:
Hello, Oleksii!
I Irejected unnecessary commits. Can you watch?
Thank you!

Reply:
Great job